universal-login-relayer/test/helpers/WalletCreator.ts

```diff
@@ -0,0 +1,69 @@
+import fetch from 'node-fetch';
+import {Wallet, Contract, utils} from 'ethers';
+import {http, HttpFunction, PublicRelayerConfig, createKeyPair, calculateInitializeSignature, computeContractAddress, TEST_GAS_PRICE} from '@universal-login/commons';
+import {encodeInitializeWithRefundData} from '@universal-login/contracts';
+import ProxyCounterfactualFactory from '@universal-login/contracts/build/ProxyCounterfactualFactory.json';
+import ENSService from '../../lib/integration/ethereum/ensService';
+
+export class WalletCreator {
+  private http: HttpFunction;
+  private relayerConfig?: PublicRelayerConfig;
+  private ensService?: ENSService;
+
+  constructor(relayerUrl: string, private wallet: Wallet) {
+    this.http = http(fetch)(relayerUrl);
+  }
+
+  async getRelayerConfig(): Promise<PublicRelayerConfig> {
+    this.relayerConfig = this.relayerConfig || (await this.http('GET', '/config')).config;
+    return this.relayerConfig!;
+  }
+
+  private async getEnsService() {
```

Comment:
`fetchEnsService`? If it is asynchronous and takes complex interactions (Http, blockchain) it should be `fetch...`. If it is a local getter it should be `get...`.